gammapy/image/models/tests/test_shapes.py

```diff
@@ -83,6 +83,15 @@ def test_delta2d_against_gauss(x_0, y_0):
     assert_allclose(values_convolved, values_gauss, atol=1E-4)
 
 
+@requires_dependency('scipy')
+@requires_data('gammapy-extra')
+def test_template2d():
+    filename = ('$GAMMAPY_EXTRA/datasets/catalogs/fermi/Extended_archive_v17'
+                '/Templates/HESSJ1841-055.fits')
+    template = Template2D.read(filename)
+    assert_allclose(template(26.7, 0), 1.1553735159851262)
+
```

Comment:
Please add an assert that exercises bounding_box and establishes the current result (to make it easier to refactor or change the code that computes it in the future).